prometheus/generic/add-target/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+haproxy_exporter_port: '9101'
+bind_exporter_port: '9119'
+ssl_exporter_port: '9219'
```

Comment:
I think we should add the `node_exporter_port: '9100'` as well.

Reply:
Yes we can. I sticked to whatever was in the original MR (from Logan) where the node exporter port was not configurable at all.

Reply:
:+1: